components/Layout.jsx

```diff
@@ -17,6 +28,7 @@ function Layout({ children }) {
   const toggleDarkMode = () => {
     document.body.classList.toggle('dark');
     document.body.classList.toggle('bg-gray-800');
+    localStorage.setItem('DARK_MODE', !darkMode);
```

Comment:
Also change the `localstorage` variable name to `theme` with the options `light` & `dark` so that in case we want to extend this functionality to other themes it can be easily done. 
Change the logic in `/components/Layout.jsx` file accordingly.